performance/performance_tuning.rst

```diff
@@ -340,6 +340,22 @@ When you're running the above pgbench benchmarks on a moderately sized Citus clu
 * Avoid closing connections between INSERT statements. This avoids the overhead of connection setup.
 * Remember that column size will affect insert speed. Rows with big JSON blobs will take longer than those with small columns like integers.
 
+Another trick is staging data temporarily in an `unlogged table <https://www.postgresql.org/docs/current/static/sql-createtable.html#SQL-CREATETABLE-UNLOGGED>`_. These are tables which are not backed by the Postgres write-ahead log. This makes them faster for inserting rows, but not suitable for long term data storage. You can use an unlogged table as a place for incoming data, and later manipulate the data and copy it to permanent tables.
```

Comment:
This doesn't seem to fit in the insert/update checklist. Should we create a new section like "Staging Data Temporarily" or something like that? (And maybe place it after the Insert/Update latency section).